# -*- coding: utf-8 -*-

'''модуль с общим описанием дизайна пользовательского интерфейса'''

from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_MainWindow(object):
    '''класс, содеражщий общее описание дизайна пользовательского интерфейса'''
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(968, 600)
        self.centralwidget = QtWidgets.QWidget(MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.verticalLayoutWidget = QtWidgets.QWidget(self.centralwidget)
        self.verticalLayoutWidget.setGeometry(QtCore.QRect(580, 470, 241, 101))
        self.verticalLayoutWidget.setObjectName("verticalLayoutWidget")
        self.verticalLayout = QtWidgets.QVBoxLayout(self.verticalLayoutWidget)
        self.verticalLayout.setContentsMargins(0, 0, 0, 0)
        self.verticalLayout.setObjectName("verticalLayout")
        self.red_acc = QtWidgets.QSlider(self.verticalLayoutWidget)
        self.red_acc.setOrientation(QtCore.Qt.Horizontal)
        self.red_acc.setObjectName("red_acc")
        self.verticalLayout.addWidget(self.red_acc)
        self.green_acc = QtWidgets.QSlider(self.verticalLayoutWidget)
        self.green_acc.setOrientation(QtCore.Qt.Horizontal)
        self.green_acc.setObjectName("green_acc")
        self.verticalLayout.addWidget(self.green_acc)
        self.blue_acc = QtWidgets.QSlider(self.verticalLayoutWidget)
        self.blue_acc.setOrientation(QtCore.Qt.Horizontal)
        self.blue_acc.setObjectName("blue_acc")
        self.verticalLayout.addWidget(self.blue_acc)
        self.verticalLayoutWidget_2 = QtWidgets.QWidget(self.centralwidget)
        self.verticalLayoutWidget_2.setGeometry(QtCore.QRect(750, 30, 176, 95))
        self.verticalLayoutWidget_2.setObjectName("verticalLayoutWidget_2")
        self.verticalLayout_2 = QtWidgets.QVBoxLayout(self.verticalLayoutWidget_2)
        self.verticalLayout_2.setContentsMargins(0, 0, 0, 0)
        self.verticalLayout_2.setObjectName("verticalLayout_2")
        self.brws_1 = QtWidgets.QPushButton(self.verticalLayoutWidget_2)
        self.brws_1.setObjectName("brws_1")
        self.verticalLayout_2.addWidget(self.brws_1)
        self.brws_2 = QtWidgets.QPushButton(self.verticalLayoutWidget_2)
        self.brws_2.setObjectName("brws_2")
        self.verticalLayout_2.addWidget(self.brws_2)
        self.brws_s = QtWidgets.QPushButton(self.verticalLayoutWidget_2)
        self.brws_s.setObjectName("brws_s")
        self.verticalLayout_2.addWidget(self.brws_s)
        self.verticalLayoutWidget_3 = QtWidgets.QWidget(self.centralwidget)
        self.verticalLayoutWidget_3.setGeometry(QtCore.QRect(20, 30, 640, 360))
        self.verticalLayoutWidget_3.setObjectName("verticalLayoutWidget_3")
        self.verticalLayout_3 = QtWidgets.QVBoxLayout(self.verticalLayoutWidget_3)
        self.verticalLayout_3.setContentsMargins(0, 0, 0, 0)
        self.verticalLayout_3.setObjectName("verticalLayout_3")
        self.verticalLayoutWidget_4 = QtWidgets.QWidget(self.centralwidget)
        self.verticalLayoutWidget_4.setGeometry(QtCore.QRect(820, 470, 127, 101))
        self.verticalLayoutWidget_4.setObjectName("verticalLayoutWidget_4")
        self.verticalLayout_4 = QtWidgets.QVBoxLayout(self.verticalLayoutWidget_4)
        self.verticalLayout_4.setContentsMargins(0, 0, 0, 0)
        self.verticalLayout_4.setObjectName("verticalLayout_4")
        self.label = QtWidgets.QLabel(self.verticalLayoutWidget_4)
        self.label.setObjectName("label")
        self.verticalLayout_4.addWidget(self.label)
        self.label_2 = QtWidgets.QLabel(self.verticalLayoutWidget_4)
        self.label_2.setObjectName("label_2")
        self.verticalLayout_4.addWidget(self.label_2)
        self.label_3 = QtWidgets.QLabel(self.verticalLayoutWidget_4)
        self.label_3.setObjectName("label_3")
        self.verticalLayout_4.addWidget(self.label_3)
        self.horizontalLayoutWidget = QtWidgets.QWidget(self.centralwidget)
        self.horizontalLayoutWidget.setGeometry(QtCore.QRect(20, 540, 396, 31))
        self.horizontalLayoutWidget.setObjectName("horizontalLayoutWidget")
        self.horizontalLayout = QtWidgets.QHBoxLayout(self.horizontalLayoutWidget)
        self.horizontalLayout.setContentsMargins(0, 0, 0, 0)
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.comb = QtWidgets.QPushButton(self.horizontalLayoutWidget)
        self.comb.setObjectName("comb")
        self.horizontalLayout.addWidget(self.comb)
        self.src = QtWidgets.QPushButton(self.horizontalLayoutWidget)
        self.src.setObjectName("src")
        self.horizontalLayout.addWidget(self.src)
        self.save_btn = QtWidgets.QPushButton(self.horizontalLayoutWidget)
        self.save_btn.setObjectName("save_btn")
        self.horizontalLayout.addWidget(self.save_btn)
        MainWindow.setCentralWidget(self.centralwidget)

        self.retranslateUi(MainWindow)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "CombineImages"))
        self.brws_1.setText(_translate("MainWindow", "Поиск файла 1"))
        self.brws_2.setText(_translate("MainWindow", "Поиск файла 2"))
        self.brws_s.setText(_translate("MainWindow", "Папка для сохранения"))
        self.label.setText(_translate("MainWindow", "Разброс красного"))
        self.label_2.setText(_translate("MainWindow", "Разброс зеленого"))
        self.label_3.setText(_translate("MainWindow", "Разброс синего"))
        self.comb.setText(_translate("MainWindow", "Совместить"))
        self.src.setText(_translate("MainWindow", "Исходное изображение"))
        self.save_btn.setText(_translate("MainWindow", "Сохранить"))

